scripts/test-migrations.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+function run_tests_for_version() {
+    set -e  # Exit immediately if a command exits with a non-zero status
+    local VERSION=$1
+
+    # Initialize zenml with the appropriate template
+    git clone -b "release/0.43.0" https://github.com/zenml-io/template-starter
```

Comment:
shouldn't this use `VERSION` instead of 0.43.0?

Reply:
No, since we don't make a new release of the `template-starter` every time we do a ZenML release. The 0.43.0 release branch / tag is simply the latest working version of the `template-starter` template...